components/fastly_backends/main.tf

```diff
@@ -14,11 +14,39 @@ variable "papertrail_destination" {
 }
 
 locals {
-  headers = {
+  request_headers = {
+    # Provide geolocation information to web servers:
     "X-Fastly-Country-Code" = "client.geo.country_code",
     "X-Fastly-Region-Code"  = "client.geo.region",
     "X-Fastly-Postal-Code"  = "client.geo.postal_code",
   }
+
+  response_headers = {
+    # Expose geolocation information in debug headers:
+    "X-Fastly-Country-Code" = "client.geo.country_code",
+    "X-Fastly-Region-Code"  = "client.geo.region",
+    "X-Fastly-Postal-Code"  = "client.geo.postal_code",
+
+    # Ensures that we don't leak sensitive items from URLs in Referrer headers: we will
+    # send full URL if same-origin and HTTPS, origin-only (e.g. 'www.dosomething.org')
+    # when crossing domains, and nothing at all when going to HTTP.
+    "Referrer-Policy" = "\"strict-origin-when-cross-origin\""
```

Comment:
To explain the funky quotes: by default Fastly interprets these values as variable references (like `client.geo.country_code` above). To set a literal string, we provide a quoted value (which, since strings in Terraform are themselves quoted, needs an escaped `\"` set of quotes inside the string).